app/src/main/java/chat/rocket/android/layouthelper/chatroom/AbstractMessageViewHolder.java

```diff
@@ -14,6 +14,7 @@
 public abstract class AbstractMessageViewHolder extends ModelViewHolder<PairedMessage> {
   protected final RocketChatAvatar avatar;
   protected final ImageView userNotFoundAvatarImageView;
+  protected final ImageView imageViewSvgUserAvatar;
```

Comment:
My suggestion is to work with Fresco, and implement support for SVG directly on it...